src/cpu/mod.rs

```diff
@@ -1136,6 +1136,7 @@ impl Cpu {
 
         self.pc = 0x0100;
         self.set_sp(0xfffe);
+        self.cycles = 0;
```

Comment:
Reset joypad on reset as well?

Reply:
joypad wont be reset from the CPU.  Resetting will be triggered from the Gameboy perspective, so once we implement that we can implement joypad reset.